tests/test_models.py

```diff
@@ -16,7 +16,7 @@ def test_registry__repr__param(setup_instance):
     param = ln.Param
     expected_repr = textwrap.dedent("""\
     Param
-      Basic fields
+      Simple fields
```

Comment:
@Zethson @fredericenard for consistency across out "django field modeling" codebase

I fear there might be lot's of duplication across what Lukas did here, and what Fred did before in the hub.

But given we somewhat have a tradition of writing these helper classes for SA & Django ORM already for years (back then also @sunnyosun wrote some of this code), we at least seem consistent in that. :D